### PR TITLE
#1060 added weatherfile with reduced line length and a second comment

### LINKED ISSUE #1060: Problems with Ashrea Weatherfile
Thanks to the new weatherfile copy implementation of @MartinRaetz in Teaser (https://github.com/RWTH-EBC/TEASER/pull/664), we recognized that the current Ashrea weatherfile for Thermalzones\ROM\Verification is not working as expected.

With the given file, there is no simulation possible.

@MartinRaetz will figure out what could be a solution, but it seems that either the keyword ATTENTION in the comments section of the weatherfile or the length of the comments line in the weatherfile seems to make problems.

Did one of you every run into problems regarding weather files comment sections or lengths? 

For further comparison between a weatherfile which works and one which does not work, you can see here the difference between the Ashrea weatherfile and the mannheim weatherfile https://www.textcompare.org/index.html?id=5fc6807b2d59400017cf4d88

### PR DESCRIPTION
**Pull request by MichaMans**
This closes #1060 

A short review on this is much appreciated 😃

### CODE REVIEW

**Review by MartinRaetz — APPROVED** [review]

File: AixLib/Resources/weatherdata/ASHRAE140.mos [hunk 1]
```diff
@@ -5,8 +5,9 @@ double tab1(8760,30)
...
 #HOLIDAYS/DAYLIGHT SAVINGS,No,0,0,0
-#COMMENTS 1,"Custom/User Format -- WMO#107290; Bundesinstitut fuer Bau-, Stadt- und Raumforschung im Bundesamt fuer Bauwesen und Raumordnung. 1) DWD are the original author of the DTRY dataset on which these EPWs are based. 2) The EPWs were converted by Climate.OneBuilding.Org. Neither the DWD nor the BBSR were responsible for the conversion, and they disclaim all liability associated with the use of the converted DTRY EPW data set. ATTENTION: So, far, it's only the header defining the location of the ASHRAE 140 weather data set. The rest is DEU_BW_Mannheim_107290_TRY2010_12_Jahr_BBSR."
+#COMMENTS 1,"Custom/User Format -- WMO#107290; Bundesinstitut f�r Bau-, Stadt- und Raumforschung im Bundesamt f�r Bauwesen und Raumordnung. 1) DWD are the original author of the DTRY dataset on which these EPWs are based. 2) The EPWs were converted by Climate.OneBuilding.Org. Neither the DWD nor the BBSR were responsible for the conversion, and they disclaim all liability associated with the use of the converted DTRY EPW data set."
```
Comment (DaJansenGit) [inline AixLib/Resources/weatherdata/ASHRAE140.mos:8]: Do we want to move umlauts from ue -> ü? Apart from this the changes are looking good ;)

**Review by DaJansenGit — APPROVED** [review]

File: AixLib/Resources/weatherdata/ASHRAE140.mos (hunk 1 above)
Comment (MichaMans) [reply AixLib/Resources/weatherdata/ASHRAE140.mos:8]: I dont know. If it throws no errors, why not?

**Comment by MichaMans** [comment]
@MartinRaetz @DaJansenGit could you please approve once more, as the previews checks were not passing :)

**Review by MartinRaetz — APPROVED** [review]
Thank you!

File: AixLib/Resources/weatherdata/ASHRAE140.mos (hunk 1 above)
Comment (mwetter) [reply AixLib/Resources/weatherdata/ASHRAE140.mos:8]: Did you check if this is a problem with the data reader from the Modelica Standard Library? Can you reproduce this with a simple problem using only the MSL, and then inform the MSL development if the problem indeed is caused by the MSL?
For me, the weather data reader works fine on Linux with OCT and Dymola, but I heard about problems on Windows with long lines.

Another fix may be to change the Java weather data converter to break long lines.

Comment (MichaMans) [reply AixLib/Resources/weatherdata/ASHRAE140.mos:8]: @mwetter Thanks for having a look at it. As far as I digged into it now it seems a problem with the getTimeSpan c function. Maybe the BUFSIZ differs between unix and windows systems and therefore leads to problems. Should I open a issue in the modelica-ibpsa for further investigation?

Comment (mwetter) [reply AixLib/Resources/weatherdata/ASHRAE140.mos:8]: @MichaMans : Yes, please open an issue on `modelica-ibpsa` and add a test case that fails on your system. Then we can see what a fix would be.

**Pull request merged**